### PR TITLE
ADEN-2009 Check next provider after Evolve hop

### PR DESCRIPTION
**Pull request by drets**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java [hunk 1]
```diff
@@ -47,16 +44,9 @@ public void verifyEvolveCall() {
...
-    WebElement evolveDiv = providers.get(0);
-    Boolean evolveIframePresent = adSlot.findElements(
-        By.cssSelector("iframe[src*=\'" + EVOLVE_DFP_URL + "\']")).size() > 0;
-    assertFalse(evolveIframePresent);
-    WebElement evolveHopScript = evolveDiv.findElement(
-        By.cssSelector("script[type='text/javascript']"));
-    String evolveHopMessage = String.format("window.evolve_hop('%s')", slotName);
-    assertStringContains(evolveHopMessage, evolveHopScript.getAttribute("src"));
+  public void verifyEvolveHoppedInSlot(String slotName, String nextProviderSrc) {
+    String nextProviderSlotCSS = String.format("div[id*=\'%s/%s\']", nextProviderSrc, slotName);
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java]: I'm going to nitpick a little bit ;) Consider going with camel-case: `nextProviderSlotCss`

**Comment by nandy-andy** [comment]
Except one small variable-name nitpicking the changes looks good for me. :+1: for more red than green :japanese_ogre:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestEvolveAds.java [hunk 2]
```diff
@@ -55,10 +55,11 @@ public void testEvolveCall_NZ(String wikiName, String article) {
       groups = {"TestEvolveAds"},
       dataProvider = "evolveHopTestPage"
   )
-  public void testEvolveHop_CA(String wikiName, String article, String slotName) {
+  public void testEvolveHop_CA(String wikiName, String article, String slotName,
```
Comment (nandy-andy) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestEvolveAds.java]: Consider going with one line or split it into four for better readability:
```java
public void testEvolveHop_CA(
  String wikiName,
  String article,
  String slotName,
  String nextProviderSrc
) {
```

**Comment by nandy-andy** [comment]
Except nitpicking about variable-name and code formatting, the changes look good for me. :+1: for more red than green :japanese_ogre:

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java (hunk 1 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java]: :+1:

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestEvolveAds.java (hunk 2 above)
Comment (drets) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestEvolveAds.java]: Sounds interesting :dancer:

**Pull request merged**